ci/build-all-steps.yml

```diff
@@ -31,6 +31,11 @@ steps:
 - script: npm install
   displayName: npm install
 
+# Install nvm tool
+- script: |
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
```

Comment:
we should add a toolcheck to make.js ensure nvm 0.39.1 is installed .  Ideally we'd print a message with instructions e.g. nvm not installed, installed with : "curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash" so it's self-documenting

Reply:
I added one here: https://github.com/microsoft/azure-pipelines-tasks/blob/99a1753d5e6a61d1fd484c2a695762a0e1c17df4/make.js#L193, it'll fail with error messages if tool not found.

Reply:
We need to communicate this well.  Probably best to print a message in make.js with instructions if nvm is not installed how to install it.  Also, would be good to post a message to channels e.g. Pipelines